docs/api-doc-markdown.md

```diff
@@ -15,6 +15,8 @@ Documents use the following markdown header fields that are enclosed by a line `
 
 `title`: The title of your document. If this field is not present, the document's `title` will default to its `id`.
 
+`description`: The description of your document which will become the `<meta name="description" content="..."/>` and `<meta property="og:description" content="..."/>` in `<head>`. If this field is not present, it will default to the first line of the contents.
```

Comment:
```suggestion
`description`: The description of your document which will become the description meta tag. If this field is not present, it will default to the first line of the contents.
```

Reply:
I prefer to keep it explicit